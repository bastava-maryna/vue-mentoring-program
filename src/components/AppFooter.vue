<script lang="ts" setup>
import NetflixLogo from "@/components/NetflixLogo.vue"
</script>

<template>
  <div class="bg-netflix-middle-gray h-14 flex items-center justify-center">
    <NetflixLogo class="h-6" />
  </div>
</template>
