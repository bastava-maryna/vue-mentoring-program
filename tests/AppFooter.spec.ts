import {shallowMount, VueWrapper} from "@vue/test-utils";
import {describe, expect, test} from "vitest";
import AppFooter from "./src/components/AppFooter.vue";

describe("AppFooter.vue", (): void => {
    test("render NetflixLogo component", (): void => {
        const wrapper: VueWrapper = shallowMount(AppFooter)

        expect(wrapper.text()).toMatchSnapshot()
        expect(wrapper.findComponent({name: "NetflixLogo"}).exists()).toBe(true)
    });
})