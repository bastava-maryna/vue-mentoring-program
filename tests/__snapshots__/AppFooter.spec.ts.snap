// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppFooter.vue > render NetflixLogo component 1`] = `""`;
